### PR TITLE
NXDRIVE-2838: Fix Release build

### PR DESCRIPTION
**Pull request by poojadaine**
NXDRIVE-2838: Fix Release builld

### CODE REVIEW

File: tools/release.sh [hunk 1]
```diff
@@ -28,7 +28,8 @@ create() {
 
     echo ">>> [${release_type} ${drive_version}] Creating the tag"
     git tag -f -a "${release_type}-${drive_version}" -m "Release ${drive_version}"
-    git push -f origin "${release_type}-${drive_version}"
+    #git push -f origin "${release_type}-${drive_version}"
```
Comment (mr-shekhar) [inline tools/release.sh:31]: is this req.?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -199,8 +198,8 @@ jobs:
 
       - name: Setup git
         run: |
-          git config user.email "maintainers-python+github@nuxeo.com"
-          git config user.name "GitHub-CI action (Release)"
+          git config user.email "drive@hyland.com"
```
Comment (mr-shekhar) [inline .github/workflows/release.yml:201]: use these as var, as they are used in several places.

**Review by mr-shekhar — CHANGES_REQUESTED** [review]
Also update other required files

**Review by mr-shekhar — CHANGES_REQUESTED** [review]
Also update other required files

File: tools/release.sh (hunk 1 above)
Comment (swetayadav1) [reply tools/release.sh]: Updated code

File: .github/workflows/release.yml (hunk 2 above)
Comment (swetayadav1) [reply .github/workflows/release.yml]: Done

**Review by mr-shekhar — APPROVED** [review]
👍

**Review by gitofanindya — APPROVED** [review]

**Pull request merged**